README.md

```diff
@@ -305,7 +305,7 @@ Community contributions are very welcome, especially:
   - plugin implementation contributions (e.g. secure-peripheral device plugins)
   - daemon implementation improvements (e.g. request data marshalling structures)
 
-Please get in touch via IRC (#jollamobile@freenode) or email if you
+Please get in touch via IRC (#sailfishos@oftc.net) or email if you
```

Comment:
This changes the name of the network to an address, if that is intentional then nevermind. If not, OFTC is written in capital letters.

Reply:
You could also change the link to an ircs:// link than the address would be fine.